### PR TITLE
fix: Review Dialect: Arabic (Python)

### PR DESCRIPTION
**Pull request by OussamaSALAHOUELHADJ**
##  Review Dialect: Arabic (Python)

This pull request (PR) is made in reference to:  #80 

## Description
Hi, while reviewing the Arabic translation for Python, I found it challenging to do such a translation. Often, there is no one word that describes a function, and in other cases, there is no word we can use, so we must use the meaning instead.
I recommend fixing the nature of the worlds either to be a noun or verb (and some rules for the verbs: passive verbs, for example), so that it becomes easier, consistent,  more organized and easier to understand. 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: locale/ar.yml [hunk 1]
```diff
@@ -1249,110 +1249,110 @@ __TRAIT__: ""
 # ----------------------
 # << Keywords >>
   False: خطأ‫"‬" #✅
```
[the remark this answers is not in the document]
Comment (OussamaSALAHOUELHADJ) [reply locale/ar.yml:1251]: Oh Ok i will work on that

File: locale/ar.yml [hunk 2]
```diff
@@ -1249,110 +1249,110 @@ __TRAIT__: ""
...
-  None: "متغير لا يحمل أي قيمة" #✅
+  None: "متغير لا يحمل أي قيمة" #❓ بلا قيمة
```
[the remark this answers is not in the document]
Comment (OussamaSALAHOUELHADJ) [reply locale/ar.yml:1252]: I think it is a bit hard or it is actually after the comment "#" (it appears with gray color), but what's making it appear that way is that Arabic is RTL language.

File: locale/ar.yml [hunk 3]
```diff
@@ -1249,110 +1249,110 @@ __TRAIT__: ""
...
-  raise: "كلمة مفتاحية لإنشاءأو إرجاع استثناء" #❓ raise: "إرجاع استثناء"
-  return: "إرجاع قيمة معينة" #❓ return: "إرجاع"
+  raise: "كلمة مفتاحية لإنشاءأو إرجاع استثناء" #🤔 raise: "إثارة استثناء" or رفع استثناء
+  return: "إرجاع قيمة معينة" #❓ return: "أرجع"
   try: " حاول، غالبا ما تستخدم في بداية كتلة استثناءات " #❓ try: "حاول"
-  while: "بيان تدوير" #❓ while: "حلقة while"
+  while: "بيان تدوير" #❓ بينما is a great fit
   with: "مع" #✅
-  yield: "إرجاع كائن مولًد لا يحفظ في الذاكرة" #🤔❓ yield: "إرجاع"
+  yield: "إرجاع كائن مولًد لا يحفظ في الذاكرة" #🤔 yield: "تحقيق" is like fulfillment/execution it describes the act of finishing from something and it's ready to be out(in the case of books for example: ready to be published)
```
[the remark this answers is not in the document]
Comment (OussamaSALAHOUELHADJ) [reply locale/ar.yml:1285]: Thank you so much.

**Comment by OussamaSALAHOUELHADJ** [comment]
Hey @madiedgar , Thanks for your change requests. I have made some changes and implemented the requested suggestions. Please review.

**Review by madiedgar — APPROVED** [review]
Looks incredible @OussamaSALAHOUELHADJ 👍 Thanks so much for implementing those changes and for your quality contribution.

**Comment by madiedgar** [comment]
Thanks, @OussamaSALAHOUELHADJ for your contribution! Really enjoyed working with you 👍

**Comment by OussamaSALAHOUELHADJ** [comment]
> Looks incredible @OussamaSALAHOUELHADJ 👍 Thanks so much for implementing those changes and for your quality contribution.

Thank you so much @madiedgar for your compliment 👊🏻 😄 

> Thanks, @OussamaSALAHOUELHADJ for your contribution! Really enjoyed working with you 👍

Hey, it was my pleasure working on such a great project, I'm always here to help 💛

**Pull request merged**